bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -54,14 +54,15 @@ FreeboxOS binding has the following configuration parameters:
 
 ### API bridge
 
-| Parameter Label               | Parameter ID      | Description                                            | Required | Default              |
-|-------------------------------|-------------------|--------------------------------------------------------|----------|----------------------|
-| Freebox Server Address        | apiDomain         | The domain to use in place of hardcoded Freebox ip     | No       | mafreebox.freebox.fr |
-| Application Token             | appToken          | Token generated by the Freebox Server.                 | Yes      |                      |
-| Network Device Discovery      | discoverNetDevice | Enable the discovery of network device things.         | No       | false                |
-| Background Discovery Interval | discoveryInterval | Interval in minutes - 0 disables background discovery  | No       | 10                   |
-| HTTPS Available               | httpsAvailable    | Tells if https has been configured on the Freebox      | No       | false                |
-| HTTPS port                    | httpsPort         | Port to use for remote https access to the Freebox Api | No       | 15682                |
+| Parameter Label               | Parameter ID        | Description                                                   | Required | Default              |
+|-------------------------------|---------------------|---------------------------------------------------------------|----------|----------------------|
+| Freebox Server Address        | apiDomain           | The domain to use in place of hardcoded Freebox ip            | No       | mafreebox.freebox.fr |
+| Application Token             | appToken            | Token generated by the Freebox Server.                        | Yes      |                      |
+| Network Device Discovery      | discoverNetDevice   | Enable the discovery of network device things.                | No       | false                |
+| Background Discovery Interval | discoveryInterval   | Interval in minutes - 0 disables background discovery         | No       | 10                   |
+| HTTPS Available               | httpsAvailable      | Tells if https has been configured on the Freebox             | No       | false                |
+| HTTPS port                    | httpsPort           | Port to use for remote https access to the Freebox Api        | No       | 15682                |
+| Websocket Reconnect Interval  | wsReconnectInterval | Disconnection interval frequency - 0 disables websocket usage | No       | 60                   |
```

Comment:
Please mention the unit in the description, it is not obvious that this is a value in minutes.

Reply:
Done